### PR TITLE
Add a note about inability to run production builds

### PR DESCRIPTION
**Pull request by lbarcziova**
packit/packit-service#848

### CODE REVIEW

File: content/docs/configuration.md [hunk 1]
```diff
@@ -351,6 +351,9 @@ With this configuration, you'll get builds in all stable fedora releases
 Create a SRPM and submit an RPM build
 to [Fedora Koji](https://koji.fedoraproject.org/koji/) build system.
 
+At the moment it is not possible to run production (non-scratch) builds from upstream.
+For more info, please see [the following issue](https://pagure.io/releng/issue/9801).
```
Comment (lachmanfrantisek) [inline content/docs/configuration.md:355]: I see a small naming issue in our setup...

We use name *production-build* for koji builds in general and koji uses *production-build* for non-scratch ones..;(

What about this to make it really clear what we mean by this?

```suggestion
At the moment it is not possible to run non-scratch production builds from upstream.
For more info, please see [the following issue](https://pagure.io/releng/issue/9801).
```

You can also append a sentence like "Scratch builds work fine."

**Review by lachmanfrantisek — COMMENTED** [review]
Thank you!

File: content/docs/configuration.md (hunk 1 above)
Comment (lbarcziova) [reply content/docs/configuration.md]: Ok, changed now

**Review by TomasTomecek — APPROVED** [review]

**Review by lachmanfrantisek — APPROVED** [review]
Thanks for such quick PR!

**Pull request merged**